### ISSUE TITLE
Abnormal behavior when resizing

### ISSUE THREAD

**Issue by khjde1207** [karvulf/flutter-reorderable-grid-view#91]
https://github.com/karvulf/flutter-reorderable-grid-view/assets/18232924/1d107c6f-9e6a-4c90-a73f-5f114227a018

These days, phones with changing screen sizes are being released, such as the Google Pixel Fold.

When the screen size changes, it behaves abnormally.

**Comment by karvulf** [comment]
Hello @khjde1207 
I see why this happens, I will try to find a fix for that

**Comment by khjde1207** [comment]
thank you !
```
 LayoutBuilder(
            builder: (context, constraints) {
                return     ReorderableBuilder.builder(
                    key: Key(
                        "${gridViewKey.toString()} ${constraints.maxWidth}"),
....
..
```

I handled this temporarily by changing the height based on the width.

**Comment by kevinsun-dev** [comment]
Bumping this issue, I'm using @khjde1207's hotfix for now, but once you attach a scrollcontroller, it throws an error about scrollcontroller mounting twice.

**Comment by karvulf** [comment]
This issue should be fixed and will come with the version `5.3.1` @kevinsun-dev @khjde1207
